CHANGELOG.md

```diff
@@ -209,7 +209,7 @@ Released on February 23, 2022.
 
 - Authentication with tokens has been removed; use API keys instead. - [#4643](https://github.com/PrefectHQ/prefect/pull/4643)
 - Python 3.6 is no longer supported; use Python 3.7+ instead. - [#5136](https://github.com/PrefectHQ/prefect/pull/5136)
-- Flow `Environment`s have been removed; use `RunConfig`s instead. - [#5072](https://github.com/PrefectHQ/prefect/pull/5072), [docs](https://docs.prefect.io/orchestration/flow_config/upgrade.html)
+- Flow `Environment`s have been removed; use `RunConfig`s instead. - [#5072](https://github.com/PrefectHQ/prefect/pull/5072), [docs](https://docs.prefect.io/v2/orchestration/flow_config/upgrade.html)
```

Comment:
Many, if not all, of the places that I see `v2`, it should in fact be `v1`, no?

Reply:
Yes you are correct. I'll fix these up